src/pybind/mgr/localpool/module.py

```diff
@@ -48,6 +52,19 @@ class Module(MgrModule):
         },
     ]
 
+    COMMANDS = [
+        {
+            "cmd": "affinity create pool "
+                   "name=poolname,type=CephString,req=true "
+                   "name=region,type=CephString,req=true "
+                   "name=pd,type=CephString,req=true "
+                   "name=sd,type=CephString,req=true "
+                   "name=td,type=CephString,req=true",
```

Comment:
can we give these args real names?  they show up in the help description

Reply:
primary_domain
secondary_domain
tertiary_domain

is that good?

Reply:
that works!

an alternative might be to make a single argument ``domain`` and set n=N, so that they can do something other than n=3.  i don't have strong feelings either way